crates/forge/tests/cli/lint.rs

```diff
@@ -530,6 +531,54 @@ Compiler run successful!
 "#]]);
 });
 
+forgetest!(can_fail_on_lints, |prj, cmd| {
+    prj.wipe_contracts();
+    prj.add_source("ContractWithLints", CONTRACT).unwrap();
+
+    // [OUTPUT: WARN + NOTE]
+    prj.update_config(|config| {
+        config.lint.fail_on = foundry_config::lint::FailOn::Note;
+        config.lint.severity = Default::default();
+    });
+
+    cmd.forge_fuse().arg("lint").assert_failure();
+
+    // [OUTPUT: WARN + NOTE]
+    prj.update_config(|config| {
+        config.lint.fail_on = foundry_config::lint::FailOn::Warning;
+        config.lint.severity = Default::default();
+    });
+    cmd.forge_fuse().arg("lint").assert_failure();
+
+    // [OUTPUT: WARN + NOTE]
+    prj.update_config(|config| {
+        config.lint.fail_on = foundry_config::lint::FailOn::Never;
```

Comment:
isn't this the default, so we can just move first and only `cmd.forge_fuse().arg("lint").assert_success();`?